modules/core/src/main/java/org/eclipse/imagen/media/opimage/AffineBicubicOpImage.java

```diff
@@ -320,7 +320,7 @@ private void byteLoop(RasterAccessor src,
                         float_fracx = fracx;
                         float_fracy = fracy;
                         frac_xx = float_fracx * (1.0F - float_fracx);
-                        frac_yy = float_fracx * (1.0F - float_fracy);
+                        frac_yy = float_fracy * (1.0F - float_fracy);
```

Comment:
Comparing with https://github.com/geosolutions-it/jai-ext/blob/master/jt-affine/src/main/java/it/geosolutions/jaiext/affine/AffineBicubicOpImage.java#L4400

As the codebases have diverged and I do not see a similar fix in place.